config/dev.exs

```diff
@@ -35,6 +35,11 @@ config :ueberauth, Ueberauth,
     cognito: {Screens.Ueberauth.Strategy.Fake, []}
   ]
 
+config :sentry,
```

Comment:
I believe that this config file is only used when `Mix.env()` returns `:dev` which should only happen when working locally. So this tag won't be used as long as local development is not connected to Sentry. Might be better to set `tags` in `config.exs` the same way you set `environment_name`.